File: comms/push-notifications/credentials/update-credential/update-credential.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.push_notifications.credentials.update(
    credential_id="comms_credential_01h9krwprkeee8fzqspvwy6nq8",
    is_default=True,
)

File: comms/push-notifications/credentials/update-credential/update-credential.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\PushNotifications\Credentials\Requests\CredentialsUpdateRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->credentials->update(
    'comms_credential_01h9krwprkeee8fzqspvwy6nq8',
    new CredentialsUpdateRequest([
        'isDefault' => true,
    ]),
);
